build.gradle

```diff
@@ -67,7 +67,7 @@ protobuf {
 
 dependencies {
     compile 'com.google.protobuf:protobuf-lite:3.0.1'
-    compile 'com.android.support:support-v4:27.1.0'
+    compile 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
that's not the right dependency, based on the imports it looks like it only needs
```
compile 'androidx.core:core:1.0.0'
```
because of these 2 classes:
androidx.core.view.ViewCompat
androidx.core.view.accessibility.AccessibilityNodeInfoCompat

ViewPager is also mentioned in a comment, but it doesn't need to be a dependency.

Reply:
@caseyburkhardt

Reply:
I'm assuming `androidx.legacy:legacy-support-v4` was used here, as that's exactly what you get frin the automated migration/refactoring from Studio.  Swapped this with `androidx.core:core:1.0.0` and things seem fine locally, so will move us over.

Reply:
[Done](https://github.com/google/Accessibility-Test-Framework-for-Android/commit/9d26064bb2b056cad58e0f83a34defce9ec8e63f)

Reply:
Yep, that's the equivalent to the old support library, but one of the benefits of AndroidX is more granular modularity, don't pull in what we don't need. The automated migration is just a tool, not magic :), it is the first step towards a brave new world. Sadly we always need to work more on getting things fully onto AndroidX. It is especially important for a library authors to reduce the pain of users finding the right version combination. Thank you for fixing it!

Reply:
@caseyburkhardt Is a v3.1.1 release planned to have this dependency change?

Reply:
We'll fold this (and a few pending updates to other dependency versions) into the binaries we push for 3.1 in the coming days.